localstack/runtime/analytics.py

```diff
@@ -37,7 +37,6 @@
     "REQUIRE_PRO",
     "SKIP_INFRA_DOWNLOADS",
     "SQS_ENDPOINT_STRATEGY",
-    "USE_SINGLE_REGION",
```

Comment:
I think we should still keep track of the usage of the old variables (this way we can reason if it's still trying to be used, even though it does not have any effects anymore).

Reply:
Restored in 46efde0a7